app/services/publish_service_creation.rb

```diff
@@ -76,6 +76,8 @@ def no_service_output?
     send_by_email.blank? || (send_by_email.present? && service_email_output.blank?)
   end
 
+  delegate :verified?, to: :from_address, prefix: true
```

Comment:
using the `prefix: true` means that it will be called by using `from_address_verified?` instead of just `verified?`. is that intentional?

Reply:
yep, we are only doing this in the view where we only need to know if form address verified. I don't mind either way - happy to change it back to a specific method if you have a preference?